site.yml

```
@@ -21,460 +21,460 @@ Documents:
 
       - How to run multiple Minio servers with Træfɪk:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/multiple-minio-servers-with-traefik.md
         Slug: how-to-run-multiple-minio-servers-with-traef-k
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to run multiple MinIO servers with Træfɪk - Cookbook/Recipe
+        SEO Description: This cookbook will explain how you can setup multiple MinIO instances via Docker which you can access on different subdomains through Træfɪk.
+        SEO Keywords: Træfɪk, reverse proxy, golang
 
       - How to use AWS SDK for Go with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-go-with-minio.md
         Slug: how-to-use-aws-sdk-for-go-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for Go with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk-go is the official AWS SDK for the Go programming language. In this recipe we will learn how to use aws-sdk-go with MinIO server
+        SEO Keywords: AWS SDK for Go, Golang
 
       - How to use AWS SDK for Java with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-java-with-minio.md
         Slug: how-to-use-aws-sdk-for-java-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for Java with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk-java is the official AWS SDK for the Java programming language. In this recipe we will learn how to use aws-sdk-java with MinIO server
+        SEO Keywords: AWS SDK for Java 
 
       - How to use Paperclip with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/how-to-use-paperclip-with-minio.md
         Slug: how-to-use-paperclip-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use Paperclip with MinIO Server - Cookbook/Recipe
+        SEO Description: Paperclip is intended as an easy file attachment library for ActiveRecord. In this recipe you will learn how to configure MinIO as an object storage backend for Paperclip.
+        SEO Keywords: Paperclip, ActiveRecord
 
       - How to use AWS SDK for .NET with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-dotnet-with-minio.md
         Slug: how-to-use-aws-sdk-for-net-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for .NET with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk-dotnet is the official AWS SDK for the .NET Framework. In this recipe we will learn how to use aws-sdk-dotnet with MinIO server.
```

Similar to my other comment for the PHP SDK, we might want to have something the following, as the first sentence doesn't show the purpose of the document directly. Once again, please feel free to dismiss if that's not the case.

```suggestion
        SEO Description: In this recipe we will learn how to use aws-sdk-dotnet, the official AWS SDK for the .NET Framework, with MinIO server.
```